knowledge_repo/app/app.py

```diff
@@ -292,6 +293,17 @@ def append_repo(self,name,uri):
         self.repository = knowledge_repo.KnowledgeRepository.append_for_uri(name,uri,temp)
         #self.db_update_index(check_timeouts=False, force=True )
         return self.repository
+    
+    def get_kr_list(self): 
+        host=request.host
+        resp = requests.get("https://%s/api/project"%(host),cookies=request.cookies)
+        krs_total = []
+        for item in resp.json():
```

Comment:
@abhirathb I guess you will have to multiprocess it somehow, as the number of requests will increase with the number of projects.

Reply:
What is the running time of this whole for loop?